### ISSUE TITLE
URL 'https://zhouserver.research.chop.edusesameData/idatSignature.rda': status was 'SSL connect error'

### ISSUE THREAD

**Issue by YoannPa** [zwdzwd/sesame#70]
Hi, 

I tried to re-execute a script I wrote few months ago, and for some reason it is not working anymore.
I don't understand why and did not find any information about this issue.

Here is the command:
`betas <- openSesame(IDATprefixes, BPPARAM = MulticoreParam(7))`
Where `IDATprefixes` is a character vector containing all my IDAT prefixes.
Here is the complete error message:
```
URL 'https://zhouserver.research.chop.edusesameData/idatSignature.rda': status was 'SSL connect error'
Erreur : BiocParallel errors
  element index: 1, 2, 3, 4, 5, 6, ...
  first error: objet 'idatSignature' introuvable
```

sessionInfo():
```
R version 4.0.5 (2021-03-31)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 16.04.7 LTS

Matrix products: default
BLAS:   /usr/lib/openblas-base/libblas.so.3
LAPACK: /usr/lib/libopenblasp-r0.2.18.so

locale:
 [1] LC_CTYPE=fr_FR.UTF-8       LC_NUMERIC=C               LC_TIME=fr_FR.UTF-8        LC_COLLATE=fr_FR.UTF-8     LC_MONETARY=fr_FR.UTF-8   
 [6] LC_MESSAGES=fr_FR.UTF-8    LC_PAPER=fr_FR.UTF-8       LC_NAME=C                  LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=fr_FR.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
 [1] grid      stats4    parallel  stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] DTrsiv_0.0.24                           RnBeads_2.8.1                           plyr_1.8.6                             
 [4] methylumi_2.36.0                        minfi_1.36.0                            bumphunter_1.32.0                      
 [7] locfit_1.5-9.4                          iterators_1.0.13                        foreach_1.5.1                          
[10] Biostrings_2.58.0                       XVector_0.30.0                          SummarizedExperiment_1.20.0            
[13] MatrixGenerics_1.2.1                    FDb.InfiniumMethylation.hg19_2.2.0      org.Hs.eg.db_3.12.0                    
[16] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2 GenomicFeatures_1.42.3                  AnnotationDbi_1.52.0                   
[19] reshape2_1.4.4                          scales_1.1.1                            Biobase_2.50.0                         
[22] illuminaio_0.32.0                       matrixStats_0.61.0                      limma_3.46.0                           
[25] gridExtra_2.3                           gplots_3.1.1                            ggplot2_3.3.5                          
[28] fields_12.5                             viridis_0.6.2                           viridisLite_0.4.0                      
[31] spam_2.7-0                              dotCall64_1.0-1                         ff_4.0.5                               
[34] bit_4.0.4                               cluster_2.1.2                           MASS_7.3-54                            
[37] GenomicRanges_1.42.0                    GenomeInfoDb_1.26.7                     IRanges_2.24.1                         
[40] S4Vectors_0.28.1                        data.table_1.14.2                       BiocParallel_1.24.1                    
[43] sesame_1.8.12                           sesameData_1.8.10                       rmarkdown_2.11                         
[46] ExperimentHub_1.16.1                    AnnotationHub_2.22.1                    BiocFileCache_1.14.0                   
[49] dbplyr_2.1.1                            BiocGenerics_0.36.1                    

loaded via a namespace (and not attached):
  [1] splines_4.0.5                 digest_0.6.28                 htmltools_0.5.2               fansi_0.5.0                  
  [5] magrittr_2.0.1                memoise_2.0.0                 tzdb_0.2.0                    readr_2.0.2                  
  [9] annotate_1.68.0               siggenes_1.64.0               askpass_1.1                   prettyunits_1.1.1            
 [13] colorspace_2.0-2              blob_1.2.2                    rappdirs_0.3.3                xfun_0.26                    
 [17] dplyr_1.0.7                   crayon_1.4.1                  RCurl_1.98-1.5                genefilter_1.72.1            
 [21] GEOquery_2.58.0               survival_3.2-13               glue_1.4.2                    gtable_0.3.0                 
 [25] zlibbioc_1.36.0               DelayedArray_0.16.3           wheatmap_0.1.0                Rhdf5lib_1.12.1              
 [29] maps_3.4.0                    HDF5Array_1.18.1              DBI_1.1.1                     rngtools_1.5.2               
 [33] Rcpp_1.0.7                    xtable_1.8-4                  progress_1.2.2                mclust_5.4.7                 
 [37] preprocessCore_1.52.1         httr_1.4.2                    RColorBrewer_1.1-2            ellipsis_0.3.2               
 [41] reshape_0.8.8                 pkgconfig_2.0.3               XML_3.99-0.8                  utf8_1.2.2                   
 [45] DNAcopy_1.64.0                tidyselect_1.1.1              rlang_0.4.12                  later_1.3.0                  
 [49] munsell_0.5.0                 BiocVersion_3.12.0            tools_4.0.5                   cachem_1.0.6                 
 [53] generics_0.1.1                RSQLite_2.2.8                 evaluate_0.14                 stringr_1.4.0                
 [57] fastmap_1.1.0                 yaml_2.2.1                    knitr_1.34                    bit64_4.0.5                  
 [61] beanplot_1.2                  scrime_1.3.5                  caTools_1.18.2                purrr_0.3.4                  
 [65] randomForest_4.6-14           nlme_3.1-153                  doRNG_1.8.2                   sparseMatrixStats_1.2.1      
 [69] mime_0.12                     nor1mix_1.3-0                 xml2_1.3.2                    biomaRt_2.46.3               
 [73] compiler_4.0.5                rstudioapi_0.13               curl_4.3.2                    interactiveDisplayBase_1.28.0
 [77] tibble_3.1.5                  stringi_1.7.5                 lattice_0.20-44               Matrix_1.3-4                 
 [81] multtest_2.46.0               vctrs_0.3.8                   rhdf5filters_1.2.1            pillar_1.6.4                 
 [85] lifecycle_1.0.1               BiocManager_1.30.16           bitops_1.0-7                  httpuv_1.6.3                 
 [89] rtracklayer_1.50.0            R6_2.5.1                      promises_1.2.0.1              KernSmooth_2.23-20           
 [93] codetools_0.2-18              gtools_3.9.2                  assertthat_0.2.1              rhdf5_2.34.0                 
 [97] openssl_1.4.5                 withr_2.4.2                   GenomicAlignments_1.26.0      Rsamtools_2.6.0              
[101] GenomeInfoDbData_1.2.4        hms_1.1.1                     quadprog_1.5-8                tidyr_1.1.4                  
[105] base64_2.0                    DelayedMatrixStats_1.12.3     shiny_1.6.0
```
Thank you very much in advance for your help.
Best.

**Comment by zwdzwd** [comment]
Hi Yoann, It looks like a known bug fixed after 1.8 (all the external server retrieval is then avoided). Any chance you could update to the most latest version (1.14.2) and see if the error is gone?

**Comment by YoannPa** [comment]
Hi,
Okay I just checked sesame documentation on Bioconductor and it seems that I would need to upgrade the outdated install of R in order to install the latest version of Sesame.
Since I managed to make it work on another PC with the latest version of R (and latest version of Sesame), I will just close the issue: the bug is already fixed.
So the answer is: Update R.
Sorry for the trouble !
